Source/JavaScriptCore/API/JSAPIGlobalObject.h

```diff
@@ -70,8 +69,9 @@ class JSAPIGlobalObject final : public JSGlobalObject {
     JSValue loadAndEvaluateJSScriptModule(const JSLockHolder&, JSScript *);
 
 private:
+    static const GlobalObjectMethodTable* globalObjectMethodTable();
     JSAPIGlobalObject(VM& vm, Structure* structure)
-        : Base(vm, structure, &s_globalObjectMethodTable)
+        : Base(vm, structure, globalObjectMethodTable())
     { }
```

Comment:
If this constructor is private:, can it be moved into the .cpp file?

Reply:
It can, yes. In fact I moved it in an earlier version of the patch but started thinking I was making too many changes.